src/service-worker.ts

```diff
@@ -10,31 +9,31 @@ const ASSETS = `cache${timestamp}`;
 const toCache = shell.concat(files);
 const cached = new Set(toCache);
 
-self.addEventListener("install", (event) => {
+self.addEventListener("install", (event: any /* ExtendableEvent */) => {
```

Comment:
I think I'd played with this a little and had to add `webworker` to the `tsconfig`. I think was able to just do `event: ExtendableEvent` then

```
     "compilerOptions": {
          "lib": [
               "webworker",
```

Reply:
653fd12